reducers/gotPlayer/player.js

```diff
@@ -28,3 +31,19 @@ export default (state = initialState, action) => {
       return state;
   }
 };
+
+export const player = {
```

Comment:
In this case we have to add it here and add/change the component.  That's two places